### DISCUSSION TITLE
Installing EOmaps on Fedora !

### DISCUSSION THREAD

**Discussion by captcha1** [raphaelquast/EOmaps#67] [Q&A]
FYI, in Fedora, before doing 'pip install eomaps' I first installed : 

`dnf install python3-mapclassify python3-cairosvg python3-xmltodict`

Of course, I had previously installed the usual suspects, eg : 

`dnf install python3-cartopy python3-xarray`

and

`dnf install python3-geopandas python3-rasterio`

The way I often determine which packages need to be install w/ dnf is to do "pip install --no-user eomaps" and see what it wants to install and then "dnf search mumble" to see if it exists in the Fedora distro.

**Comment by raphaelquast** [comment]
Hey, thanks a lot for sharing this! I'll point to here in the docs in case people run into troubles installing EOmaps with pip.
